docs/core/versions/binding.md

```diff
@@ -0,0 +1,170 @@
+
+Occasionally, ASP.NET packages are deployed via a web host rather than with a published application. In those cases, the web host should correctly configure their environments based on published guidance such that all supported ASP.NET applications run correctly.
+
+Developers should remember the following rules:
+
+- Runtimes are searched for in the following order (the first found is loaded):
+  - Highest patch version (check that the minimum runtime path version is satisfied, otherwise error)
+  - Highest minor version
+- If an acceptable runtime is not found, the application does not run and an error message explains why.
+
+## Publish a self-contained application
+
+You can publish an application as a **self-contained distribution**. This approach includes .NET Core. Self-contained distributions do not have a dependency on runtime environments. Runtime binding occurs at publishing time, not run-time.
+
+The publishing process will select the latest patch version of the given runtime family. For example, `dotnet publish` will select .NET Core 2.0.4 if it is the latest patch version in the .NET Core 2.0 runtime family, independent of the existence of earlier patch versions or minimum runtime patch version values. The publishing process differs from `dotnet build` runtime selection for two reasons:
+
+- The application doesn't specify a hosting environment.
+- The developer generates a final configuration for the application (that can't be practically modified). The best runtime choice is the latest installed runtime patch version. It's the same version used for development and it has the latest (per the machine) security and reliability fixes.
+
+The `runtimeframeworkversion` element will override the default version policy if that element is present in the project file.
+
+It is an error if the minimum version specified for an application is not satisfied. `dotnet publish` binds to latest runtime patch version (within a given major.minor version family). `dotnet publish` does not support the roll-forward semantics of `dotnet run`.
+
+## Publish an application with a specific .NET Core version
+
+`dotnet publish` accepts a version to use for publishing for both self-contained and framework-dependent apps. This is primarily for CI/CD scenarios or other times when you must publish an application, but can't update source code.
+
+You publish with the latest runtime patch version for the given target framework specifying `--latest`. This argument has the same default runtime binding behavior for publishing self-contained applications. See the following example:
+
+``` console
+dotnet publish -c release -o app --latest
```

Comment:
what is the value of the other switches here. Either drop or write out as the reader may not remember what c and o stand for

Reply:
dropping this section